src/Game.java

```diff
@@ -0,0 +1,115 @@
+import java.util.*;
+
+public class Game {
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+    private static final int MAX_ERRORS = 8;
+
+    private Scanner scanner = new Scanner(System.in);
+
+    private String wordToFind;
+    private char[] guessedWord;
```

Comment:
Why guessed word is a array of chars? What do you store here?

Reply:
A word that indicates the user's progress during the game, that is, first the letters of the word converted to "*", which are successively discovered

Reply:
got it